app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryimage/EntryImageActivity.java

```diff
@@ -182,10 +184,32 @@ public void showInvalidEntryErrorMessage() {
     }
 
     @Override
-    public void showApiErrorMessage(int error_api) {
+    public void showApiErrorMessage() {
```

Comment:
gonna port my comment over to here to make things easier. Is there anyway to make this more reusable?